health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/GeoPode.js

```
@@ -134,64 +182,65 @@ const GeoPode = () => {
                             onClick={() => {
                                 // setShowPopUp(true);
                             }}
-                        /> 
+                        />
                     </div>
-                    <div style={{height:"2rem"}}></div>
+                    <div style={{ height: "2rem" }}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)